composer.json

```diff
@@ -8,9 +8,9 @@
     "license": "MIT",
 
     "require": {
-        "php": ">=5.5.0",
+        "php": "<=5.5.99",
```

Comment:
5.3 and 5.4 are not supported, anyone who wants to use an old version, can switch to 0.x versions.